addons/payment_authorize/models/payment.py

```diff
@@ -104,7 +133,7 @@ def _authorize_form_get_tx_from_data(self, data):
         reference, trans_id, fingerprint = data.get('x_invoice_num'), data.get('x_trans_id'), data.get('x_MD5_Hash')
         if not reference or not trans_id or not fingerprint:
             error_msg = 'Authorize: received data with missing reference (%s) or trans_id (%s) or fingerprint (%s)' % (reference, trans_id, fingerprint)
-            _logger.error(error_msg)
+            _logger.info(error_msg)
```

Comment:
This is probably not even needed since the `raise ValidationError` should log a full traceback and the message as well.